data-url/src/forgiving_base64.rs

```diff
@@ -17,6 +32,18 @@ pub enum DecodeError<E> {
     WriteError(E),
 }
 
+impl<E> std::fmt::Display for DecodeError<E> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::InvalidBase64(inner) => write!(f, "base64 not valid: {:?}", inner),
```

Comment:
Since we still support Rust 1.36, you'll have to write out the full enum name here.

Reply:
Could this please be merged now that the MSRV is 1.51?